department-of-veterans-affairs/caseflow
# Hotfix/appeals 21696

Resolves [Caseflow | Hearings: Pending confirmation emails resulting in partially converted appeal](https://jira.devops.va.gov/browse/APPEALS-21696)

# Description
Converting a non-virtual hearing to virtual fails on occasion.

## Acceptance Criteria

## Testing Plan
1. Create a non-virtual hearing. Ensure every party has an email address entered. Convert to virtual. Verify that the links we're created & are visible.

2. Create a non-virtual hearing. Ensure every party has an email address entered except for the appellant. Convert to virtual. Verify that the links we're created & are visible.

# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
